src/main/java/pages/OwnerInformationPage.java

```diff
@@ -59,6 +60,14 @@ public void clickAddPetButton(){
     }
 
 
+    /**
+     * This method checks if the new visit has been added
+     */
+    public void verifyAddedNewVisit() {
+        WebElement newvisit = driver.findElement(By.cssSelector("table.table:nth-of-type(2) table.table-condensed td"));
```

Comment:
The project is based on page factory. We better move this to the top of the class.